cdap-docs/_common/_source/index.rst

```diff
@@ -104,18 +104,6 @@ CDAP capabilities include *CDAP Pipelines* and *Metadata Management*.
   - |dev-man-at-black|_ Adding a custom logback, best practices for CDAP development,
     class loading in CDAP, configuring program resources
 
-.. |pipelines-black| replace:: **Pipelines:**
```

Comment:
didn't we want the link to be there though? Or will it be only under Developer's manual? I'm fine either way.